SQRLPlatformAwareInstaller/ViewModels/VersionSelectorViewModel.cs

```diff
@@ -62,9 +62,9 @@ private  void InitObj(string platform ="WINDOWS")
                 CachePolicy = new System.Net.Cache.RequestCachePolicy(System.Net.Cache.RequestCacheLevel.NoCacheNoStore)
             };
 
-            wc.Headers.Add("User-Agent", GitHubHelper.UserAgent);
+            wc.Headers.Add("User-Agent", GitHubHelper.SQRLInstallerUserAgent);
 
-            this.Releases =  GitHubHelper.GetReleases().Result;
+            this.Releases = await GitHubHelper.GetReleases();
```

Comment:
@alexhauser  in this case I was using GithubHelper.GetRelease().Result because we don't want the UI to draw up empty. When I did it with async prior it would draw up render blank and then fill in when the release call came through.

In this instance I think its better to call Result like we had instead of making this call async the blank installer may cause confusion.

The issue you ran into with the API limitation i've ran into before they limit the number of calls per hour per IP address in general outside of developing it shouldn't be a major issue.

Thoughts?

Reply:
Yes, I think I got tripped up by the rate limiting error. Without the await, the app would just not do anything. It would also not hit any breakpoint etc. But I didn't know about the rate limiting at that point, I only found out about that later. 

So yes - I will revert it to how it was before. Thanks for the input, Jose!

Reply:
BTW @alexhauser  while you are developing you can Authenticate the call to github which gives you basically unlimited rate.
To do that head over to 
https://github.com/settings/tokens

Generate a new Personal access token I give mine all the permissions cause I'm lazy
Then anywhere the web client is used you can add an authorization header like this
![image](https://user-images.githubusercontent.com/420837/78457415-0b8a7d80-7678-11ea-8518-618eb08adc46.png)

NOte: Don't keep the token in the final commit / push cause 
a) it would be a bad security issue for you
b) Github immediately notices and disables your token

Reply:
@josegomez, I've just reverted it to the way it was before, but now again I can't get the installer to work correctly that way. It looks like it hits an endless loop when hitting next. 

May I suggest that I merge it with the await and then you look at it and make it work correctly again? THANKS!

Reply:
Sounds good merge away I'll take a look